app/run.py

```diff
@@ -232,6 +232,7 @@ def callback(pubsub_message):
                 'policy_id': evaluation.policy_id,
                 'remediable': evaluation.remediable,
                 'resource_data': resource.to_dict(),
+                'resource_labels': resource.labels,
```

Comment:
looking ahead to the next feature, would we rather see this?
```
'labels': {
    'resource': resource.labels,
    'project': project.labels,
}
```

Reply:
I'm happy with this change if the answer is no

Reply:
Joe and I talked about this, he's not sure it's feasible to get ancestry labels in RTE anyhow.  That might need to be picked up by consumes of these logs.